applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -333,8 +333,9 @@ To connect to an nRF91 Series DK with a PC
 1. Verify that ``UART_0`` is selected in the application.
    It is defined in the default configuration.
 
-2. Use LTE Link Monitor to connect to the development kit.
-   See :ref:`lte_connect` for instructions.
+2. Use `nRF Connect Serial Terminal`_ to connect to the development kit.
+   See :ref:`serial_terminal_connect` for instructions.
+   You can also use the `Cellular Monitor`_ app to open the nRF Connect Serial Terminal.
```

Comment:
I don't think that asset_tracker_v2_description.rst has a previous mention of Cellular Monitor either, but you are correct it does show up here out of the blue.

We use Cellular Monitor for collecting modem traces via UART, but there is no reference on the page, on how to actually achieve this. We only have a link to "Sending traces over UART on an nRF91 Series DK" on this page, but I am wondering, should we also have a link to how-to collect the traces with cellular modem, or at least mention it with trace collection.

I think that it would be good to mention Cellular Monitor and the possibility to open the nRF Connect Serial Terminal through it on this page, as that is the easiest way to launch it when you are getting modem trace from the application.

Reply:
+1 on more info about how to use Cellular Monitor rather than not mentioning it. Using Cellular Monitor and Serial Terminal in combo would give the user more information about what is happening, so I think it's worth informing about. Does opening Serial Terminal from Cellular Monitor give any extra benefits? I'm not sure it's worth mentioning that detail, but more focus on the combination of the two applications. Perhaps rephrasing the sentence would help?

```suggestion
   By using the `Cellular Monitor`_ application in combination with nRF Connect Serial Terminal, you'll better see how the modem responds to the different modem commands.
```